sigar_interface.go

```diff
@@ -106,6 +106,7 @@ const (
 
 type ProcState struct {
 	Name      string
+	UserId    string
```

Comment:
When I read `UserId` I am thinking of a numeric value like a unix UID. I think I would name this `Username` like Golang does in their [User](https://golang.org/pkg/os/user/#User) struct. But that's just one person's opinion, I'm open to hearing others' opinions.